src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/UpdateSitesManager.java

```diff
@@ -156,8 +156,8 @@ public void doUpdate(StaplerRequest req, StaplerResponse rsp, @Sites List<Update
         shouldNotContainDuplicatedIds(newSitesList);
         shouldNotContainBlankIds(newSitesList);
 
-        Jenkins.getInstance().getUpdateCenter().getSites().replaceBy(newSitesList);
-        Jenkins.getInstance().getUpdateCenter().save();
+        Jenkins.getActiveInstance().getUpdateCenter().getSites().replaceBy(newSitesList);
+        Jenkins.getActiveInstance().getUpdateCenter().save();

```

Comment:
<BugInstance type='NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE' priority='Normal' category='STYLE' message='Possible null pointer dereference in jp.ikedam.jenkins.plugins.updatesitesmanager.UpdateSitesManager.doUpdate(StaplerRequest, StaplerResponse, List) due to return value of called method' lineNumber='159'/>
    <BugInstance type='NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE' priority='Normal' category='STYLE' message='Possible null pointer dereference in jp.ikedam.jenkins.plugins.updatesitesmanager.UpdateSitesManager.doUpdate(StaplerRequest, StaplerResponse, List) due to return value of called method' lineNumber='160'/>